src/kaleidoscope/hid.cpp

```diff
@@ -51,6 +51,9 @@
 namespace kaleidoscope {
 namespace hid {
 
+static byte mod_flags{0};
+static byte mod_flags_allowed{0};
```

Comment:
FWIW, you don't have to initialize `static` variables to zero, they're guaranteed to be anyway.